en/preprocessor.adoc

```diff
@@ -8,7 +8,7 @@
 The Red preprocessor is a dialect of Red, enabling transformation of Red source code using a specific layer on top of regular Red language. Transformations are achieved by inlining preprocessor keywords (called `directives`) inside Red source code. These directives will be processed:
 
 * when the Red source code is compiled
-* when the Red source code is run by `do` function with a file argument
+* when the Red source code is runned by `do` function with a file argument
```

Comment:
I used `runned by` as a synonym for `launched by` here, perhaps it should be... `evaluated with`?